tests/test_drop_plugin.py

```
@@ -157,6 +157,37 @@ async def run_test():
+        """
+        mock_connect.return_value = self.mock_meshtastic_client
+
+        packet = {
+            "fromId": "!12345678",
+            "decoded": {
+                "portnum": TEXT_MESSAGE_APP,
+                "text": "please process !drop Embedded message",
+            },
+        }
+
+        async def run_test():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)